backend/src/main/java/gov/cdc/usds/simplereport/db/model/auxiliary/TestResultDeliveryPreference.java

```diff
@@ -0,0 +1,7 @@
+package gov.cdc.usds.simplereport.db.model.auxiliary;
+
+public enum TestResultDeliveryPreference {
+  SMS,
+  EMAIL,
```

Comment:
Email is listed in this enum, but there's no corresponding logic on the backend to implement it. I'd consider either documenting that here, or adding a TODO in TestOrderService to implement email notification when it's available.

Reply:
Good call, I will add a TODO since we will probably implement that sometime in the future :+1: